config/product/logging.js

```diff
@@ -314,7 +314,7 @@ export function init(store) {
           delimiter:                 { type: 'string' },
           delimiter_pattern:         { type: 'string' },
           label_delimiter:           { type: 'string' },
-          multiline:                 { type: 'arrray[string]' },
+          multiline:                 { type: 'array[string]' },
```

Comment:
??

Reply:
`array[string]` is the proper type.. the variation with `rrr` also isn't used anywhere else, telling me this is a typo